documentation/basics/headless.md

```diff
@@ -228,7 +229,7 @@ For developers and power-users, you can run your own ilastik-dependent python sc
     # Mac
     $ ./ilastik-1.3.2-OSX.app/Contents/ilastik-release/bin/python -c "import ilastik; print ilastik.__version__"
     1.3.2
-
+Check out [this example](https://github.com/ilastik/ilastik/blob/master/examples/example_python_client.py) on how to use ilastik as a python module
```

Comment:
We plan on changing the way to use ilastik as a python module completely and don't want people to start building projects now, relying on the old way.
```suggestion
```

Reply:
Oh I'm sorry I didn't know that. I am currently building a project relying on that and it took some time for me to find this example.